TheMET/TheMET/DataModels/Extentions.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  itemApearence.swift
```

Comment:
Please check for a better file-name. `Extensions.swift` is too generic.
The common practice for such fundamental(UIKit) extensions is to:
1) Do a single extension per file. So you'd need 3 extensions
2) Name files as "ComponentName+additionalFunctionalityName.swift". In your case "UILabel+apply.swift", "UIButton+apply.swift", "UINavigationItem+apply.swift" would work perfectly

Reply:
fixed